### PR TITLE
[#3836] Update testing support.

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
=====

There is a but where when using sudo, codecov token is not correctly passed.

Changes
=======

Pass the resolved variable... in this way it should also work with older sudo versions

As a drive by I have updated the paver.sh and also updated the python test to support the future 'random' pluging which will soon land on server.

How to try and test the changes
===============================

reviewers: @brunogola 

check that changes make sense

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

File: release-notes.rst [hunk 1]
```diff
@@ -4,6 +4,20 @@ chevah-brink release notes
 Here are the release notes for past brink version.
 
 
+Version 0.57.0, 2017-01-31
+--------------------------
+
+* Update paver.sh from python-package to add support for macOS 10.12.
+* Test runner can not be setup with a list of default nose arguments.
```
Comment (brunogola) [inline release-notes.rst]: Test runner can **not** -> **now**

**Comment by brunogola** [comment]
changes-approved

**Comment by adiroiban** [comment]
thanks... this was released and is already used by chevah/server

**Pull request closed without merging**